### PR TITLE
travis: run on node 6 through node 14

### PR DESCRIPTION
**Pull request by danielbeardsley**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,3 +2,5 @@ language: node_js
 node_js:
   - 6
   - 8
+  - 10
+  - 12
```
Comment (djmetzle) [inline .travis.yml:6]: Any reason to not include current?
```suggestion
  - 12
  - 14
```

Comment (danielbeardsley) [reply .travis.yml:6]: TIL about v14!!

Comment (djmetzle) [reply .travis.yml:6]: It's been current for a while. 12 went LTS in October:

 https://nodejs.org/en/about/releases/

**Comment by djmetzle** [comment]
CR :call_me_hand:

**Comment by danielbeardsley** [comment]
Passes on all the things!

image

**Pull request merged**